code/modules/mob/living/carbon/human/life/handle_shock.dm

```diff
@@ -1,8 +1,5 @@
 //Refer to life.dm for caller
 
-/mob/living/carbon/human/feels_pain()
```

Comment:
You haven't changed any of the code which calls `feels_pain()`, and the next proc up is `/mob/living/proc/feels_pain()`, which just returns `TRUE`.
As far as I can tell, this just makes all human mobs feel pain. `handle_shock.dm` doesn't seem to check for `NO_PAIN` at all.

I'd recommend having `/mob/living/carbon/human/feels_pain()` check for the existence of the `NO_PAIN` flag rather than the var, instead of removing the proc as a whole.

Reply:
This was a duplicate definition
See https://github.com/vgstation-coders/vgstation13/blob/daf2a41526ee54c7a3aa23a9ba26935152244d0b/code/modules/mob/living/carbon/human/human.dm#L1663

Reply:
I really hate that BYOND just silently allows duplicate definitions.